worlds/cvcotm/items.py

```diff
@@ -0,0 +1,183 @@
+    # Map
+    iname.cleansing:      CVCotMItemData(0xE806, ItemClassification.progression),
+    iname.roc_wing:       CVCotMItemData(0xE807, ItemClassification.progression),
+    iname.last_key:       CVCotMItemData(0xE808, ItemClassification.progression_skip_balancing),
+    iname.ironmaidens:    CVCotMItemData(None, ItemClassification.progression),
+    iname.dracula:        CVCotMItemData(None, ItemClassification.progression),
+    iname.shinning_armor: CVCotMItemData(None, ItemClassification.progression),
+}
+
+action_cards = {iname.mercury, iname.venus, iname.jupiter, iname.mars, iname.diana, iname.apollo, iname.neptune,
+                iname.saturn, iname.uranus, iname.pluto}
+
+attribute_cards = {iname.salamander, iname.serpent, iname.mandragora, iname.golem, iname.cockatrice, iname.griffin,
+                   iname.manticore, iname.thunderbird, iname.unicorn, iname.black_dog}
+
+filler_item_names = [iname.heart_max, iname.hp_max, iname.mp_max]
+
+
+def get_item_names_to_ids() -> Dict[str, int]:
+    return {name: cvcotm_item_info[name].code + base_id for name in cvcotm_item_info
+            if cvcotm_item_info[name].code is not None}
+
+
+def get_item_counts(world: "CVCotMWorld") -> Dict[ItemClassification, Dict[str, int]]:
+
+    item_counts = {
+        ItemClassification.progression: {},
+        ItemClassification.progression_skip_balancing: {},
+        ItemClassification.useful: {},
+        ItemClassification.filler: {},
+    }
+    total_items = 0
+    excluded_cards = []
+
+    # If Halve DSS Cards Placed is on, determine which cards we will exclude here.
+    if world.options.halve_dss_cards_placed:
+        excluded_cards = list(action_cards.union(attribute_cards))
+        freeze_actions = [iname.mercury, iname.mars]
+        freeze_attrs = [iname.serpent, iname.cockatrice]
+        has_freeze_action = False
+        has_freeze_attr = False
+        start_card_cap = 8
+
+        start_cards = [item for item in world.options.start_inventory_from_pool.value if "Card" in item]
+
+        # Check for ice/stone cards that are in the player's starting cards. Increase the starting card capacity by 1
+        # for each card type satisfied.
+        for card in start_cards:
+            if card in freeze_actions and not has_freeze_action:
+                has_freeze_action = True
+                start_card_cap += 1
+            if card in freeze_attrs and not has_freeze_attr:
+                has_freeze_attr = True
+                start_card_cap += 1
+
+        # If we are over our starting card capacity, some starting cards will need to be removed...
+        if len(start_cards) > start_card_cap:
+            kept_start_cards = []
+            removed_start_cards = []
+
+            # Remove all but the ice/stone cards; these we'll keep no matter what.
+            for card in start_cards:
+                if card in freeze_actions + freeze_attrs:
```

Comment:
They can add extras, or at least can after https://github.com/ArchipelagoMW/Archipelago/pull/3158

Reply:
If someone were to add multiple instances of specific cards to the pool, the code will remove all instances of whichever specific cards it decides to take out. Having duplicates of the same card is pointless because you only ever need one instance to use all of the combos associated with it, and you don't ever lose cards.